app/js/disease-associations-table-directive.js

```diff
@@ -420,64 +456,69 @@ angular.module('cttvDirectives')
             //     render( new_state, old_state ); // if there are no facets, no worries, the API service will handle undefined
             // });
 
+            scope.$watchGroup(["filters", "disease", "target", "excludedtargetlist"], function (attrs) {
 
-
-            scope.$watchGroup(["filters", "disease"], function (attrs) {
                 filters = attrs[0];
                 var disease = attrs[1];
+                scope.target = attrs[2];
+                scope.excludedTargetList = attrs[3];
```

Comment:
Again, excludedTargetList is already in the scope of the directive, I think you don't need to reset it here.
In fact the same happens with scope.target